app/uploaders/item_photo_uploader.rb

```diff
@@ -0,0 +1,55 @@
+# encoding: utf-8
+
+class ItemPhotoUploader < CarrierWave::Uploader::Base
+
+  # Include RMagick or MiniMagick support:
```

Comment:
This won't hold up the PR, but in general, delete the generated comments once you are done with setting up this file